plugins/task/src/index.ts

```diff
@@ -178,6 +180,8 @@ export interface KanbanTemplateSpace extends Doc {
   name: IntlString
   description: IntlString
   icon: AnyComponent
+  relatedIssueTemplatesComponent?: AnyComponent
```

Comment:
This should be configurable via Mixin, so any kind of template configuration will be available.

Reply:
Done